docs/admin.md

```diff
@@ -0,0 +1,96 @@
+| configmap-files.yaml  | corev1.ConfigMap    | No            |0.02|Backstage config file inclusions from configMap   |
+| configmap-envs.yaml   | corev1.ConfigMap    | No            |0.02|Backstage env variables from configMap    |
+| secret-files.yaml     | corev1.Secret       | No            |0.02|Backstage config file inclusions from Secret   |
+| secret-envs.yaml      | corev1.Secret       | No            |0.02|Backstage env variables from Secret    |
+| dynamic-plugins.yaml  | corev1.ConfigMap    | No            |0.02|dynamc-plugins config *    |
+| dynamic-plugins-      | corev1.ConfigMap    | No            |0.01|dynamc-plugins config *    |
+|configmap.yaml         |                     |               |    | |
+| backend-auth-         | corev1.ConfigMap    | No            |0.01|backend auth config    |
+|configmap.yaml         |                     |               |    | |
+
+
+
+NOTES: 
+ - Mandatory means it is needed to be present in either (or both) Default and CR Raw Configuration.
+ - dynamic-plugins.yaml is a fragment of app-config.yaml provided with RHDH/Janus-IDP. The reason it mentioned separately is specific way it provided (via mounting to dedicated initContainer)  
+ - items marked as version 0.01 are not supported in version 0.02 
+### Operator Bundle configuration 
+
+With Backstage Operator's Makefile you can generate bundle descriptor using *make bundle* command
+
+Along with CSV manifest it generates default-config ConfigMap manifest, which can be modified and applied to Backstage Operator
+TODO: is kubectl sufficient or there are some other tools?
+
+### Kustomize deploy configuration
+
+Make sure use the current context in your kubeconfig file is pointed to correct place, change necessary part of your config/manager/default-config or just replace some of the file(s) with yours and run
+``
+make deploy
+``
+
+### Direct ConfigMap configuration
+
+You can change default configuration by directly changing the default-config ConfigMap with kubectl like:
+
+ - retrieve the current `default-config` from the cluster
+
+``
+kubectl get -n backstage-system configmap default-config > my-config.yaml
+``
+
+- modify the file in your editor of choice
+
+- apply the updated configuration to your cluster
+
+``
+  kubectl apply -n backstage-system -f my-config.yaml
+``
+
+It has to be re-applied to the controller's container after being reconciled by kubernetes processes.
+
+
+### Use Cases
+
+#### Airgapped environment
+
+Creating Backtage CR, the Operator will try to create Backstage Pod, deploying:
+- Backstage Container from the image, configured in *(deployment.yaml).spec.template.spec.Containers[].image*
+- Init Container (applied for RHDH/Janus-IDP configuration, usually the same as Backstage Container)
+- 
+Also, if Backstage CR configured as EnabledLocalDb it will create Database (PGSQL) Container, configured in *(db-deployment.yaml).spec.template.spec.Containers[].image*
```

Comment:
```suggestion
Also, if Backstage CR configured with *EnabledLocalDb*,  it will create a PostgreSQL container pod, configured in *(db-deployment.yaml).spec.template.spec.Containers[].image*
```